build_runner_core/test/asset_graph/graph_test.dart

```diff
@@ -513,14 +513,6 @@ void main() {
             expect(globNode.globNodeState!.inputs, contains(coolAssetId));
             expect(globNode.globNodeState!.results, contains(coolAssetId));
             await checkChangeType(ChangeType.REMOVE);
-            expect(
```

Comment:
I don't understand why is this removed.

Reply:
It used to be that the asset graph update would invalidate the glob node _and_ modify the result to remove the deleted node; now it just invalidates the glob node, so in this test it's not updated yet.

I have a work-in-progress PR that refactors away from using the graph to track invalidation, at which point this test goes away altogether.

Thanks.